src/GitLabApiClient/Models/Member.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Newtonsoft.Json;
+
+namespace GitLabApiClient.Models
+{
+    public sealed class Member : Assignee
```

Comment:
Member is not a different variant of Assignee, it just only has some similar properties, for now it can diverge later, because they are two orthogonal concepts, can you remove inheritance from here and just declare all assignee properties here?